XCUITests/HomePageSettingsUITest.swift

```diff
@@ -11,9 +11,9 @@ let exampleUrl = "test-example.html"
 
 class HomePageSettingsUITests: BaseTestCase {
     private func enterWebPageAsHomepage(text: String) {
-        app.textFields["HomePageSettingTextField"].tap()
-        app.textFields["HomePageSettingTextField"].typeText(text)
-        let value = app.textFields["HomePageSettingTextField"].value
+        app.textFields["HomeAsCustomURLTextField"].tap()
```

Comment:
I can't find where was the previous id...have you changed it here directly?
This id has to be changed in the [UITests](https://github.com/mozilla-mobile/firefox-ios/blob/80de754bb6e2d935aa4357540b06c4d06af15c3f/UITests/Global.swift#L675) too or we might have failures there...

Reply:
There's only one other place where I was able to find something relating to HomePageSetting (not including titles of HomeSettingPageController/HomePageSetting tests or functions called homePageSetting) and it is a string in global.swift called "HomePageSettingTextField"- it looks to be a separate definition not depending on "HomePageSetting" ID. I will link it here: https://github.com/mozilla-mobile/firefox-ios/blob/master/UITests/Global.swift @farhanpatel

Reply:
That's the same file as the link I pasted above, it is for the UI Tests, you should change it there to so that there are no failures in the UI Test suite...
I was asking because it looks like you are changing the id directly in the test, if it has not been modified in any other file and I didn't know that was doable :)

Reply:
Right? I was weirded out when it worked too.. but I just pushed a commit for changing Global.swift and we can see if it passes the test!